src/Microsoft.Azure.SignalR/HubHost/ServiceHubDispatcherOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Http.Connections;
+
+namespace Microsoft.Azure.SignalR
+{
+    //todo: public for [HttpTransportType] feature
+    /// <summary>
+    /// Options used to configure the Azure SignalR Service hub dispatcher.
+    /// </summary>
+    internal class ServiceHubDispatcherOptions
+    {
+        /// <summary>
+        /// Gets or sets a bitmask combining one or more <see cref="HttpTransportType"/> values that specify what transports the server should use to receive HTTP requests.
+        /// </summary>
+        public HttpTransportType Transports { get; set; } = HttpTransports.All
```

Comment:
is putting this into current `ServiceOptions` enough?

Reply:
In SignalR, this option is per hub, so I think the user might need to use different options for different hub.